people.html

```diff
@@ -8,24 +8,32 @@
 </head>
 
 <body>
-    <nav class="navbar navbar-expand-md navbar-dark bg-dark">
-        <div class="container-fluid">
-            <a class="navbar-brand" href="index.html">
-				Soteria Collaborators
+	<nav class="navbar navbar-expand-md">
+		<div class="container-fluid">
+			<a class="navbar-brand" href="index.html">
+				<img class="navbar-logo" src="assets/images/navbarLogo.png" />
+				<span>Soteria</span>
 			</a>
-            <ul class="navbar-nav">
-                <li class="nav-item">
-                    <a class="nav-link" href="demo.html">Demo</a>
-                </li>
-                <li class="nav-item">
-                    <a class="nav-link" href="people.html">People</a>
-                </li>
-                <li class="nav-item">
-                    <a class="nav-link" href="wiki.html">Wiki</a>
-                </li>
-            </ul>
-        </div>
-    </nav>
+			<ul class="navbar-nav">
+				<li class="nav-item">
+					<a class="nav-link" href="index.html">Home</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="demo.html">Demo</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="people.html">People</a>
+				</li>
+				<li class="nav-item">
+					<a class="nav-link" href="wiki.html">Wiki</a>
+				</li>
+				<li class="nav-item">
+					<i class="fa fa-search search-icon"></i>
+				</li>
+			</ul>
+		</div>
+	</nav>
+
     <main class="container style-fix">
         <h1>Software</h1>
         <div class="container">
```

Comment:
I mentioned this in PR #13, but could you change some div tags to section tags wherever it makes sense? Also, is there supposed to be two container classes used here?

Reply:
When should I use section tags instead of div tags? Also, I'm not sure about the container classes.